### PR TITLE
Revert "Create the cart page and main functions of the cart #38 #41"

### PR DESCRIPTION
**Pull request by Chovv**
Reverting shopping cart features as they seem to be conflicting with the functionality of the site itself.

### CODE REVIEW

**Comment by AfandiM** [comment]
Fair, I will work on this branch, but in the meantime, we should revert this PR

**Comment by kalsij** [comment]
yes, i agree,  we should revert for now

File: store/src/components/Navigation Bar/Navigation Bar.js [hunk 1]
```diff
@@ -124,14 +124,7 @@ const Navbar = ({ setToken, userInfo, setUserInfo }) => {
                         >
                             {pages[0][0]}
                         </Button>
-                        <Button name="cart"
-                            component={Link}
-                            to={pages[4][0]}
-                            onClick={handleCloseNavMenu}
-                            sx={{ my: 2, color: 'inherit', display: 'block' }}
-                        >
-                            {pages[4][1]}
```
Comment (AfandiM) [inline store/src/components/Navigation Bar/Navigation Bar.js:133]: Array Index out of bound, the ```pages``` array only goes up to 3 indices.

This explains the error

**Review by AfandiM — APPROVED** [review]

**Review by GuardiansAscend — APPROVED** [review]
Necessary for now, will take a look at these changes as well

**Review by kalsij — APPROVED** [review]

**Pull request merged**